lib/gax.js

```diff
@@ -58,6 +58,20 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {boolean=} isLongrunning - If set to false and the call returns a
+ *   longrunning operation, the initial operation will be returned and polling
+ *   will not be performed.
+ * @property {Object=} longrunning
+ * @property {anyDecoder=} longrunning.responseDecoder - The decoder to unpack
+ *   the response message. If not provided, the response will be unpacked if
+ *   the proto descriptor of the any value is found in the longrunningDescriptor
+ *   protoDescriptorPool.
+ * @property {anyDecoder=} longrunning.metadataDecoder - The decoder to unpack
+ *   the metadata message. If not provided, the metadata will be unpacked if
+ *   the proto descriptor of the any value is found in the longrunningDescriptor
+ *   protoDescriptorPool.
```

Comment:
`responseDecoder` and `metadataDecoder` should be in `LongrunningDescriptor` instead of here -- I assume they would not change on individual method calls.